kubectl-plugin/pipeline/gc.go

```diff
@@ -44,7 +44,7 @@ func newGCCmd(client dynamic.Interface) (cmd *cobra.Command) {
 
 const (
 	conditionAnd = "and"
-	conditionOr  = "or"
+	conditionOr  = "ignoreTime"
```

Comment:
Do we still call the const as `conditionOr`?

Reply:
my bad

Reply:
see https://github.com/kubesphere-sigs/ks/pull/219/commits/621b6a0ed21d13a4b666ffe072464cbd741e8d0f